tests/unittests/test-cprofile.cpp

```diff
@@ -262,6 +262,39 @@ TEST(profile_data, TryParse_StringBufferW)
 	ASSERT_STREQ(L"", value.c_str());
 }
 
+/*!
+ * @brief ToString(KEYCODE)のテスト
+ */
+TEST(profile_data, ToString_KEYCODE)
+{
+	std::wstring str;
+	KEYCODE code;
+
+	code = 0x80;
+	str = profile_data::ToString(code);
+	ASSERT_STREQ(L"", str.c_str());
+
+	code = -1;
+	str = profile_data::ToString(code);
+	ASSERT_STREQ(L"", str.c_str());
+
+	code = 0;
+	str = profile_data::ToString(code);
+	ASSERT_STREQ(L"", str.c_str());
+
+	code = 1;
+	str = profile_data::ToString(code);
+	ASSERT_STREQ(L"\x01", str.c_str());
+
+	code = 0x61;
+	str = profile_data::ToString(code);
+	ASSERT_STREQ(L"a", str.c_str());
```

Comment:
Pull Requestのレビュー結果に関係ありませんが、アサーションの書き方について改善提案です。

```suggestion
	EXPECT_THAT(str, testing::StrEq(L"a"));
```

----

旧来の書き方(C言語前提、std::stringに.c_str()が必要。）
```c
    ASSERT_STREQ( expected, actual );
```

gmockを使った書き方(C++前提、std::stringに.c_str()は不要。）
```c++
    EXPECT_THAT( actual, testing::StrEq(expected) );
```